weave/trace_server_bindings/remote_http_trace_server.py

```diff
@@ -161,6 +224,13 @@ def _flush_calls(
             self._flush_calls(batch[split_idx:], _should_update_batch_size=False)
             return
 
+        if encoded_bytes > CLICKHOUSE_SINGLE_ROW_INSERT_BYTES_LIMIT:
+            batch = self._strip_large_values_from_batch(batch, encoded_bytes)
+
+        if len(batch) == 0:
+            return
+
+        encoded_data = Batch(batch=batch).model_dump_json().encode("utf-8")
```

Comment:
i think you are double encoding in the case that you don't need to strip

Reply:
as in: encoding twice

Reply:
yeah nice. moved second encoding to inside the `if encoded_bytes > CLICKHOUSE_SINGLE_ROW_INSERT_BYTES_LIMIT:` condition